addons/onboarding/static/src/views/form/onboarding_step_form_controller.js

```diff
@@ -52,4 +57,10 @@ export default class OnboardingStepFormController extends FormController {
     get stepConfig() {
         return { reloadAlways: false, reloadOnFirstValidation: false };
     }
+    /**
```

Comment:
I would keep a description here as you initially had.

Reply:
Ah my bad , probably remove when i copy paste.
Done